build.gradle

```diff
@@ -49,43 +36,76 @@ minecraft {
 }
 
 dependencies {
-	//to change the versions see the gradle.properties file
+	// To change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-
 	mappings "io.github.minecraft-cursed-legacy:plasma:b1.7.3-build.${project.plasma_build}"
-
 	modImplementation("io.github.minecraft-cursed-legacy:cursed-fabric-loader:${project.loader_version}") {
 		transitive false
 	}
 
 	// API. You technically don't need it, but it's extremely useful for not having to write the same code in every mod.
 	modImplementation "io.github.minecraft-cursed-legacy:cursed-legacy-api:${project.api_version}"
+
+	// PSA: Some older mods, compiled on Loom 0.2.1, might have outdated Maven POMs.
+	// You may need to force-disable transitiveness on them.
 }
 
 processResources {
-	inputs.property 'version', project.version
+	inputs.property "version", project.version
```

Comment:
Pointless change. This is a string literal. Has no inline variables. Single quotes are correct here.

Reply:
It's what the standard example mod has, I just copy-pasted its buildscript and then manually put in the cl example mod changes